yarn.lock

```diff
@@ -2898,10 +2898,10 @@
   resolved "https://registry.yarnpkg.com/@types/prop-types/-/prop-types-15.5.1.tgz#1ecf52621299e65b855374337fb11fd2d1066fc1"
   integrity sha1-Hs9SYhKZ5luFU3Qzf7Ef0tEGb8E=
 
-"@types/react-native@^0.62.2":
-  version "0.62.8"
-  resolved "https://registry.yarnpkg.com/@types/react-native/-/react-native-0.62.8.tgz#224602561f75b838ed6e3b5ea37093bb84cffd74"
-  integrity sha512-YEf0tH3xYJpQB12Vvzoy7cqPY3mvbbulTnG3G7ToKOuJuqigAN3K9NNAaxNAAm1zCj+UtObhzcaJtPRwX1z6Fw==
+"@types/react-native@^0.62.10":
```

Comment:
are the changes from 62.2 to 62.10 necessary for the switch from accessibilityStates to accessibilityState?

Reply:
We normally pull these in automatically, but this one is a little bit special.

0.62 removed `accessibilityStates` from upstream RN. We missed updating our typings for that, which meant we had some code laying around that didn't work that didn't see a compile-time error. Eloy fixed this with a recent typings update (see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44717), which by design breaks in our repo until we fix the other stuff up.